layouts/partials/event_card.html

```diff
@@ -1,3 +1,29 @@
-<div class="event_card">
-  <h2>{{ .Title }}</h2>
+<link rel="stylesheet" href="../../assets/scss/components/_event_card.scss" />
+<link rel="preconnect" href="https://fonts.gstatic.com" />
+<link
+  href="https://fonts.googleapis.com/css2?family=Nunito:wght@600&display=swap"
+  rel="stylesheet"
+/>
```

Comment:
Please move these to head.html. Right now these lines are getting appended to the dom for every event card. We only need to link the font once.

Reply:
Right, got it.